visualization/app/codeCharta/util/gameObjectsParser/gameObjectsImporter.ts

```diff
@@ -0,0 +1,138 @@
+import md5 from "md5"
+
+function isFile(names) {
+	return names.length === 0
+}
```

Comment:
I assume that names is a list of strings. This file is missing type declarations on each function parameter.
We lose our typescript advantages, if we don't specify the types here.